### ISSUE TITLE
Autostart not working Ubuntu 23.04 Cinnamon

### ISSUE THREAD

**Issue by ajb3932** [LizardByte/Sunshine#2016]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Sunshine does not autostart, the service unit works fine when ran manually but when rebooted shows as 

>      Active: inactive (dead)

### Expected Behavior

Sunshine auto starts

### Host Operating System

Linux

### Operating System Version

Ubuntu 23.04 Cinnamon

### Architecture

64 bit

### Sunshine commit or version

v0.21.0

### Package

Linux - deb

### GPU Type

Intel

### GPU Model

GTX 1080ti

### GPU Driver/Mesa Version

535

### Capture Method (Linux Only)

X11

### Config

```Shell
[Unit]
Description=Sunshine is a self-hosted game stream host for Moonlight.
StartLimitIntervalSec=500
StartLimitBurst=5
# delay initialization to after graphical-session.target has been started
After=graphical-session.target

[Service]
ExecStart=/usr/bin/sunshine
# uncomment the line below if your system is reporting session initialization prematurely
# ExecStartPre=/usr/bin/sleep 30 # wait  30s before starting sunshine
Restart=on-failure
RestartSec=5s

[Install]
# activate this unit alongside graphical-session.target
WantedBy=graphical-session.target
```

### Relevant log output

```shell
Jan 12 16:16:39 SteamPC sunshine[4859]: terminate called after throwing an instance of 'std::filesystem::__cxx11::filesystem_error'
Jan 12 16:16:39 SteamPC sunshine[4859]:   what():  filesystem error: cannot create symlink: File exists [/dev/input/event13] [/home/alex/.config/sunshine/sunshine_mouse]
Jan 12 16:16:40 SteamPC systemd[1102]: sunshine.service: Main process exited, code=dumped, status=6/ABRT
Jan 12 16:16:40 SteamPC systemd[1102]: sunshine.service: Failed with result 'core-dump'.
Jan 12 16:16:40 SteamPC systemd[1102]: sunshine.service: Consumed 2.331s CPU time.
Jan 12 16:16:46 SteamPC systemd[1102]: sunshine.service: Scheduled restart job, restart counter is at 1.
Jan 12 16:16:46 SteamPC systemd[1102]: Stopped sunshine.service - Sunshine is a self-hosted game stream host for Moonlight..
Jan 12 16:16:46 SteamPC systemd[1102]: sunshine.service: Consumed 2.331s CPU time.
Jan 12 16:16:46 SteamPC systemd[1102]: Started sunshine.service - Sunshine is a self-hosted game stream host for Moonlight..
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Sunshine version: v0.21.0
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: System tray created
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Error: Failed to create session: This hardware does not support NvFBC
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Error: Environment variable WAYLAND_DISPLAY has not been defined
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detecting monitors
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detected monitor 0: HDMI-0, connected: true
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detected monitor 1: DP-0, connected: false
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detected monitor 2: DP-1, connected: false
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detected monitor 3: DP-2, connected: false
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detected monitor 4: DP-3, connected: false
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detected monitor 5: DP-4, connected: false
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Detected monitor 6: DP-5, connected: false
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: // Testing for available encoders, this may generate errors. You can safely ignore those errors. //
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Trying encoder [nvenc]
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Screencasting with X11
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Screencasting with X11
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: SDR color coding [Rec. 601]
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Color depth: 8-bit
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Color range: [JPEG]
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Warning: [h264_nvenc @ 0x55feef38ea40] Multiple reference frames are not supported by the device
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Fatal: [h264_nvenc @ 0x55feef38ea40] Provided device doesn't support required NVENC features
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Error: Could not open codec [h264_nvenc]: Function not implemented
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Screencasting with X11
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: SDR color coding [Rec. 601]
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Color depth: 8-bit
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Color range: [JPEG]
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Error: [AVHWDeviceContext @ 0x55feef27de40] Primary context already active with incompatible flags.
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Error: Failed to create a CUDA device: Operation not supported
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Screencasting with X11
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: SDR color coding [Rec. 601]
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Color depth: 8-bit
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Info: Color range: [JPEG]
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Error: [AVHWDeviceContext @ 0x55feef325440] Primary context already active with incompatible flags.
Jan 12 16:16:46 SteamPC sunshine[4953]: [2024:01:12:16:16:46]: Error: Failed to create a CUDA device: Operation not supported
```

**Comment by ReenigneArcher** [comment]
Duplicate of #1533
